src/get_session_access_token_test.ts

```diff
@@ -32,27 +49,42 @@ Deno.test("getSessionAccessToken()", async (test) => {
     );
   });
 
-  await test.step("rejects for an expired access token", async () => {
+  await test.step("rejects for an access token which expires in less than 5 seconds", async () => {
     const sessionId = crypto.randomUUID();
     const tokens: Tokens = {
       ...genTokens(),
-      expiresIn: 0,
+      refreshToken: crypto.randomUUID(),
+      expiresIn: 3,
     };
     await setTokens(sessionId, tokens);
-    assertRejects(async () =>
+    await assertRejects(async () =>
       await getSessionAccessToken(oauth2Client, sessionId)
     );
   });
 
-  await test.step("rejects if the OAuth provider hasn't issued the access token", async () => {
+  await test.step("rejects for an expired access token", async () => {
     const sessionId = crypto.randomUUID();
     const tokens: Tokens = {
       ...genTokens(),
-      expiresIn: 60,
+      refreshToken: crypto.randomUUID(),
+      expiresIn: 0,
     };
     await setTokens(sessionId, tokens);
-    assertRejects(async () =>
+    await assertRejects(async () =>
       await getSessionAccessToken(oauth2Client, sessionId)
     );
   });
+
+  // await test.step("rejects if the OAuth provider hasn't issued the access token", async () => {
+  //   const sessionId = crypto.randomUUID();
+  //   const tokens: Tokens = {
+  //     ...genTokens(),
+  //     refreshToken: crypto.randomUUID(),
+  //     expiresIn: 60,
+  //   };
+  //   await setTokens(sessionId, tokens);
+  //   await assertRejects(async () =>
+  //     await getSessionAccessToken(oauth2Client, sessionId)
+  //   );
+  // });
```

Comment:
After adding the `await` to `assertRejects` here, the test fails because correct conditions are met to just return the token. Is there something specific this is supposed to be testing? I can't tell if there is really a way to test what the descriptions says it is testing. I'm not really sure this one makes sense to include

Reply:
Can you please open an issue? Let's tackle this separately.

Reply:
Should I leave it commented or remove the `await` in the meantime?

Reply:
Let's completely revert this block as to keep the diff quiet.